src/Aspire.Hosting.Azure.Sql/AzureSqlExtensions.cs

```diff
@@ -28,6 +28,7 @@ internal static IResourceBuilder<SqlServerServerResource> PublishAsAzureSqlDatab
             sqlServer.AssignProperty(x => x.Administrators.Sid, construct.PrincipalIdParameter);
             sqlServer.AssignProperty(x => x.Administrators.Login, construct.PrincipalNameParameter);
             sqlServer.AssignProperty(x => x.Administrators.TenantId, "subscription().tenantId");
+            sqlServer.AssignProperty(x => x.MinimalTlsVersion, "'1.3'");
```

Comment:
Why are we using 1.3 here, but https://github.com/dotnet/aspire/pull/4695 set the minimum to 1.2?

Reply:
Azure Storage doesn't support TLS 1.3 (see https://learn.microsoft.com/azure/storage/common/transport-layer-security-configure-minimum-version).

The reason I was setting 1.3 here was defaulting to the latest supported version, not minimal consistency.

Reply:
https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.SQL.MinTLS/ and https://learn.microsoft.com/en-us/azure/azure-sql/database/connectivity-settings?view=azuresql&tabs=azure-portal#minimum-tls-version says the recommended minimum is to set it to 1.2.

The learn docs even have a caution:

![image](https://github.com/user-attachments/assets/785fa643-ed68-4d27-830c-82c8831dd123)

Reply:
Cool, wasn't aware of that (I should read the docs properly 🤣).

I'll drop it to 1.2 as the min for Azure SQL

Reply:
I pushed an update to change it to 1.2. I'd like to get this into the 8.2 release, which is getting snapped from main tomorrow.

Reply:
I was just about to update myself, you beat me to it